arm-advisor/2017-03-31/swagger/advisor.json

```diff
@@ -0,0 +1,679 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2017-03-31",
+		"title": "AdvisorManagementClient",
+		"description": "REST APIs for Azure Advisor"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations": {
+			"post": {
+				"tags": [
+					"GenerateRecommendations"
+				],
+				"description": "Initiates the recommendation generation or computation process for a subscription. This operation is asynchronous. The generated recommendations are stored in a cache in the Advisor service.",
+				"operationId": "Recommendations_Generate",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "Accepted.",
+						"headers": {
+							"Location": {
+								"description": "The URL where the status of the asynchronous operation can be checked.",
+								"type": "string"
+							},
+							"Retry-After": {
+								"description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+								"type": "string"
+							}
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GenerateRecommendations": {
+						"$ref": "../examples/GenerateRecommendations.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations/{operationId}": {
+			"get": {
+				"tags": [
+					"GenerateRecommendations"
+				],
+				"description": "Retrieves the status of the recommendation computation or generation process. Invoke this API after calling the generation recommendation. The URI of this API is returned in the Location field of the response header.",
+				"operationId": "Recommendations_GetGenerateRecommendationsStatus",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"name": "operationId",
+						"in": "path",
+						"description": "The operation ID, which can be found from the Location field in the generate recommendation response header.",
+						"required": true,
+						"type": "string",
+						"format": "uuid"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"202": {
```

Comment:
Here you mention 202 which is accepted. Now, how will I poll back to get the response? Which URI to hit? Usually it is obtained from the response of the 202. But there is no response here.